source/developer-guides/configuration.rst

```diff
@@ -19,6 +24,51 @@ Internal configuration data
 External configuration data
   Platform specific data which is configurable by tools. It can be protected with user provided key.
 
+.. _Configuration Files and Configuration Flow:
+
+One firmware image can support configuration requirements of multiple platforms. To support this, Slim Bootloader solution creates DSC file and DLT file to handle board level differences.
+
+.. image:: /images/Config1.jpg
+         :width: 600
+         :alt: Install Ubuntu 1 of 5
+
+Configuration Flow
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. image:: /images/Config2.jpg
+         :width: 600
+         :alt: Install Ubuntu 1 of 5
+
+_DSC File
+^^^^^^^^^^^^
+* DSC file is the heart of all platform settings including memory, silicon, GPIO, OS boot policy, security settings etc.
+* DSC file, in general is located in project specific board folder. 
+* For example, you can find DSC file for APL platform under SblOpen\Platform\ApollolakeBoardPkg\CfgData folder.
+
+Please note that you may find many DSC files. However, only one DSC file is used for all of the platform configuration, and other sub DSC files are for special component configuration.

```

Comment:
Let us be more specific.   

Please note that you may find many DSC files. However, only CfgDataDef.dsc is the primary file used for the platform configuration, and other sub DSC files will be included by the primary DSC file to provide component specific configuration.

Reply:
Okay, will correct it.